resources/taxForms.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseListParams, TaxForm, UnitConfig, UnitResponse } from "../types"
+import { BaseResource } from "./baseResource"
+
+export class TaxForms extends BaseResource {
+    constructor(token: string, basePath: string, config?: UnitConfig) {
+        super(token, basePath + "/tax-forms", config)
+    }
+
+    public async list(params?: TaxFormsListParams): Promise<UnitResponse<TaxForm[]>> {
+        const parameters = {
+            "page[limit]": (params?.limit ? params.limit : 100),
+            "page[offset]": (params?.offset ? params.offset : 0),
+            ...(params?.accountId && { "filter[accountId]": params.accountId }),
+            ...(params?.customerId && { "filter[customerId]": params.customerId }),
+            ...(params?.taxYears && { "filter[taxYears]": params.taxYears }),
+            ...(params?.taxFormTypes && { "filter[taxFormTypes]": params.taxFormTypes })
+        }
+
+        return this.httpGet<UnitResponse<TaxForm[]>>("", { params: parameters })
+    }
+
+    public get(taxFormId: string): Promise<UnitResponse<TaxForm>> {
+        return this.httpGet<UnitResponse<TaxForm>>(`/${taxFormId}`)
+    }
+
+    public getPdf(taxFormId: string): Promise<string> {
+        return this.httpGet<string>(`/${taxFormId}/pdf`)
+    }
```

Comment:
@axshani let's add the params that we always add for file downloads (pdfs, images) like responseEncoding and responseType for consistency (see checkDeposit and checkPayments `getImage` and statements get / getBankVerification)

Reply:
done